docs/How-To_Guides/OpenStack/Cinder/encrypted-volumes.md

```diff
@@ -0,0 +1,157 @@
+| encrypted           | True                                 |
+| id                  | 33211b21-8d4f-48e9-b76f-ec73ffd19def |
+| multiattach         | False                                |
+| name                | enc_drive                            |
+| properties          |                                      |
+| replication_status  | None                                 |
+| size                | 10                                   |
+| snapshot_id         | None                                 |
+| source_volid        | None                                 |
+| status              | creating                             |
+| type                | volumes_hdd_encrypted                |
+| updated_at          | None                                 |
+| user_id             | 966ad341f4e14920b5f589f900246ccc     |
++---------------------+--------------------------------------+
+```
+
+Upon volume creation, this will create a one-off encryption key, which
+is stored in Barbican and applies to this one volume only. In other
+words, the key created for this volume will be unable to decrypt any
+other volumes except the one it was created for.
+
+
+## Retrieving a volume’s encryption key
+
+Once you have created an encrypted volume, you may retrieve a
+reference to the Barbican secret that represents its encryption
+key. You do this with the following command:
+
+```bash
+openstack volume show \
+  --os-volume-api-version 3.66 \
+  -f value \
+  -c encryption_key_id \
+  enc_drive
+```
+
+Instead of the volume name, you can of course also specify its UUID:
+
+```bash
+openstack volume show \
+  --os-volume-api-version 3.66 \
+  -f value \
+  -c encryption_key_id \
+  33211b21-8d4f-48e9-b76f-ec73ffd19def
+```
+
+
+## Deleting an encrypted volume
+
+When you decide you no longer need an encrypted volume and want to
+delete it, you can do so with the `openstack volume delete`
+command. As long as you do this with the same user account as the one
+that created the volume, this will succeed without further
+intervention.
+
+However, if you are trying to delete a volume that was created by a
+different user, you’ll run into the limitation that the *secret*
+associated with the volume is owned by that user. As a result, the
+deletion of the encrypted volume using your own user credentials will
+fail.
+
+There are two options to work around this limitation:
+
+1. You can switch to the user credentials of the user that created the
+   volume (if you have access to them), and proceed with the deletion.
+2. You can ask the user that created the volume to [add you to the
+   Access Control List (ACL) for the
+   secret](../Barbican/share-secret.md). This will enable you to read
+   the secret, and to delete the volume using your own credentials.
+
+
+## Block device encryption caveats
+
+Once a volume is configured to use encryption and is also attached to
+an instance in {{extra.brand}}, some caveats apply that you might want
```

Comment:
Is {{extra.brand}} really applicable here?

Reply:
why not? It looks okay to me.

Reply:
Yes, it's correct here.